src/app/admin/modules/data-management/components/data-graph/data-graph.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container-fluid">
+  <div class="row">
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>选择统计类型</mat-label>
+          <mat-select [formControl]="selectedGraphType">
+            <mat-option *ngFor="let statistics of statisticsType" [value]="statistics.id">{{statistics.name}}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>选择分组类型</mat-label>
+          <mat-select [formControl]="selectedGroupType">
+            <mat-option *ngFor="let groupType of secondType[selectedGraphType.value]" [value]="groupType.id">{{groupType.name}}</mat-option>
```

Comment:
Won't this break the code since you haven't initialize the `selectedGraphType`, but you access `selectedGraphType.value`.

Reply:
It's a bit of hacky to display different types of choices based on previous input, consider using `*ngIf` to determine which `<mat-select>` should be used.